### ISSUE TITLE
please please fix the bug 

### ISSUE THREAD

**Issue by meemo00** [afzafri/UiTM-Timetable-Generator#69]
please fix the bug asap, we need the timetable TT

**Comment by adibzter** [comment]
@meemo00, please refer the official one if this generator doesn't work as intended

Official: https://simsweb4.uitm.edu.my/estudent/class_timetable/index.htm

**Comment by meemo00** [comment]
yes we do refer to the official one to check classes but students wants the READY TIMETABLE which we generate here, to refer from time to time.

hopefully the admin(s) can understand and try to fix the bugs.

**Comment by adibzter** [comment]
@meemo00 I've deployed new version. Let me know if it fixed your issue or not
